bindings.cpp

```diff
@@ -2690,6 +2691,15 @@ void DeRestPluginPrivate::checkLightBindingsForAttributeReporting(LightNode *lig
     {
         return;
     }
+    
+    // Hack : Need to disable reporting for thoses devices, else It will enable a auto power off after 2mn.
+    // see https://github.com/dresden-elektronik/deconz-rest-plugin/issues/3693
+    if (lightNode->manufacturer() == QLatin1String("_TZ3000_fvh3pjaz") ||
+        lightNode->manufacturer() == QLatin1String("_TZ3000_9hpxg80k") ||
+        lightNode->manufacturer() == QLatin1String("_TZ3000_wyhuocal"))
+    {
+        return;
+    }
```

Comment:
Can you please add this small change, to make it a bit less costly :)

```
   if (lightNode->manufacturer().startsWith('_') &&
       (lightNode->manufacturer() == QLatin1String("_TZ3000_fvh3pjaz") ||
        lightNode->manufacturer() == QLatin1String("_TZ3000_9hpxg80k") ||
        lightNode->manufacturer() == QLatin1String("_TZ3000_wyhuocal")))
    {
        return;
    }
```

Reply:
Done